doc/manuals/user/v1_v2_coexistence.md

```diff
@@ -19,13 +19,15 @@ This document explains some consideration to take into account regarding such co
 ## Native JSON types
 
 NGSIv2 allows to create/update attributes (and metadata) whose values use JSON native 
-types (number, boolean, string, etc.). Unfortunately, NGSIv1 uses a JSON parser that converts 
+types (number, boolean, string, etc.). By default, NGSIv1 uses a JSON parser that converts
 numbers and boolean values to string at creation/update time. Thus, an attempt of 
-setting `A=2` using NGSIv1 will actually store `A="2"` in the Orion database.
+setting `A=2` using NGSIv1 will actually store `A="2"` in the Orion database. However,
+if is possible some degree of native types support in NGSIv1 storing using
```

Comment:
However, some degree of native types is possible in NGSIv1 ...

Reply:
Fixed in f65e5f29a1e53753c25d1c2e81ea7e3277fb241c